README.md

```diff
@@ -54,14 +53,16 @@ If you've been struggling to deploy headless browsers without running into issue
 - Error tolerant: if Chrome dies it won't.
 - Support for running and development on Apple's M1 machines
 
-### Cloud
+### Cloud-only
 
 Our [cloud accounts](https://www.browserless.io/pricing/) include all the general features plus extras, such as:
 
-- Inbuilt [residential proxy](https://www.browserless.io/blog/2023/09/24/residential-proxying/)
-- [/unblock API](https://www.browserless.io/blog/2024/02/26/unblock-api/) for avoiding detectors
-- [Hybrid automations](https://www.browserless.io/blog/2024/03/21/hybrid-automations-for-puppeteer/) for streaming login windows during scripts
-- Ability to upload and run extensions _(coming soon)_
+- Inbuilt [residential proxy](https://www.browserless.io/blog/residential-proxying/)
+- [/unblock API](https://www.browserless.io/unblock-api/) for avoiding detectors
```

Comment:
Do you mean https://docs.browserless.io/HTTP-APIs/unblock? The link in the commit give a 404 :/

Reply:
Good catch, although I was intending to link it to https://www.browserless.io/blog/unblock-api